src/web/server/src/index.ts

```diff
@@ -0,0 +1,16 @@
+console.log("Try npm run check/fix!");
+
+const longString = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer ut aliquet diam.';
+
+const trailing = 'Semicolon'
+
+            const why = 'am I tabbed?';
+
+export function doSomeStuff(withThis: string, andThat: string, andThose: string[]) {
```

Comment:
What is this

Reply:
Same

Reply:
Seems like this is something Chieh added for Typescript formatting. I'm not sure if this is still needed. @chc5

Reply:
Oh, I didn't see that the linter auto-generated this file.